packages/paste-core/primitives/box/src/index.tsx

```diff
@@ -81,6 +90,15 @@ interface BoxBaseStyleProps
   float?: FloatProperty;
   willChange?: WillChangeProperty;
   clip?: ClipProperty;
+  backgroundAttachment?: BackgroundAttachmentProperty;
+  textAlign?: TextAlignProperty;
+  textTransform?: TextTransformProperty;
+  textDecoration?: TextDecorationProperty<string>;
+  textOverflow?: TextOverflowProperty;
+  whiteSpace?: WhiteSpaceProperty;
+  wordBreak?: WordBreakProperty;
+  wordWrap?: WordWrapProperty;
+  overflowWrap?: OverflowWrapProperty;
```

Comment:
Most of these are already declared in style props.

Overflow wrap should be added to layout style props

Reply:
Goood catch. Been a minute since I touched these files. Fixed.

Reply:
Totally forgot about those being there.